common/sass/styles/btn-toggle.scss

```diff
@@ -0,0 +1,66 @@
+/* TOGGLE BUTTON */
+$slider-radius: 26px;
+$transition-time: .4s;
+$toggle-color: #0C6482;
+$travel-distance: 38px;
+
+.switch {
```

Comment:
Prefer SuitCSS classnames as mentioned before, should probably be a component style rather than a global one